.pre-commit-config.yaml

```
@@ -13,16 +13,12 @@ repos:
   - id: mixed-line-ending
   - id: trailing-whitespace
 
-- repo: https://github.com/psf/black-pre-commit-mirror
-  rev: 23.10.1
-  hooks:
-  - id: black
-
 - repo: https://github.com/astral-sh/ruff-pre-commit
   rev: v0.1.2
```

```suggestion
  rev: v0.1.3
```

It is released so maybe bump.